Empaquetage failed

Salut @charlesfleche , de retour avec deux semaines de dispo pour de l'audio-numérique sous linux, je viens d'essayer de construire la version 0.12 et l'empaquetage plante. La compilation semble bien se passer, mais les tests semblent échouer :

```
...
...
cd TestDB/ && ( test -e Makefile || /usr/lib/x86_64-linux-gnu/qt5/bin/qmake /build/lpd8editor-0.0.12/test/TestDB/TestDB.pro -nocache 'QMAKE_CFLAGS_RELEASE=-g -O2 -fdebug-prefix-map=/build/lpd8editor-0.0.12=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2' 'QMAKE_CFLAGS_DEBUG=-g -O2 -fdebug-prefix-map=/build/lpd8editor-0.0.12=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2' 'QMAKE_CXXFLAGS_RELEASE=-g -O2 -fdebug-prefix-map=/build/lpd8editor-0.0.12=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2' 'QMAKE_CXXFLAGS_DEBUG=-g -O2 -fdebug-prefix-map=/build/lpd8editor-0.0.12=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2' QMAKE_LFLAGS_RELEASE=-Wl,-z,relro QMAKE_LFLAGS_DEBUG=-Wl,-z,relro QMAKE_STRIP=: PREFIX=/usr -o Makefile ) && make -f Makefile check
make[3]: Entering directory '/build/lpd8editor-0.0.12/test/TestDB'
/build/lpd8editor-0.0.12/test/TestDB/target_wrapper.sh  ./TestDB 
make[3]: Entering directory '/build/lpd8editor-0.0.12/test/TestCommands'
/build/lpd8editor-0.0.12/test/TestCommands/target_wrapper.sh  ./TestCommands 
********* Start testing of TestCommands *********
Config: Using QtTest library 5.7.1, Qt 5.7.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 6.3.0 20170124)
********* Start testing of TestDB *********
Config: Using QtTest library 5.7.1, Qt 5.7.1 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 6.3.0 20170124)
QWARN  : TestDB::initTestCase() QSqlDatabase: QSQLITE driver not loaded
QWARN  : TestCommands::initTestCase() QSqlDatabase: QSQLITE driver not loaded
QWARN  : TestCommands::initTestCase() QSqlDatabase: available drivers: 
QWARN  : TestDB::initTestCase() QSqlDatabase: available drivers: 
QWARN  : TestCommands::initTestCase() QSqlDatabase: an instance of QCoreApplication is required for loading driver plugins
QWARN  : TestDB::initTestCase() QSqlDatabase: an instance of QCoreApplication is required for loading driver plugins
PASS   : TestDB::initTestCase()
PASS   : TestCommands::initTestCase()
QWARN  : TestDB::test_idsSequence() QSqlQuery::exec: database not open
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::exec: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to initialize db ""
QWARN  : TestDB::test_idsSequence() Failed to initialize db ""
QWARN  : TestDB::test_idsSequence() QSqlQuery::prepare: database not open
QWARN  : TestDB::test_idsSequence() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_idsSequence() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
FAIL!  : TestDB::test_idsSequence() Compared values are not the same
   Actual   (createProgram()): -1
   Expected (1)              : 1
   Loc: [tst_db.cpp(43)]
QWARN  : TestDB::test_fromSysex() QSqlQuery::exec: database not open
QWARN  : TestDB::test_fromSysex() Failed to initialize db ""
QWARN  : TestDB::test_fromSysex() QSqlQuery::prepare: database not open
QWARN  : TestDB::test_fromSysex() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_fromSysex() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_fromSysex() QSqlQuery::prepare: database not open
QWARN  : TestDB::test_fromSysex() Failed to update program from sysex:  "Driver not loaded Driver not loaded"
FAIL!  : TestDB::test_fromSysex() 'fromSysex(id, allzero_sysex)' returned FALSE. ()
   Loc: [tst_db.cpp(51)]
QWARN  : TestDB::test_createProgram(All default) QSqlQuery::exec: database not open
QWARN  : TestDB::test_createProgram(All default) Failed to initialize db ""
QWARN  : TestDB::test_createProgram(All default) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(All default) Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(All default) Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
FAIL!  : TestDB::test_createProgram(All default) Compared values are not the same
   Actual   (createProgram(name, sysex, id)): -1
   Expected (resId)                         : 1
   Loc: [tst_db.cpp(121)]
QWARN  : TestDB::test_createProgram(All specified) QSqlQuery::exec: database not open
QWARN  : TestDB::test_createProgram(All specified) Failed to initialize db ""
QWARN  : TestDB::test_createProgram(All specified) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(All specified) Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(All specified) Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
FAIL!  : TestDB::test_createProgram(All specified) Compared values are not the same
   Actual   (createProgram(name, sysex, id)): -1
   Expected (resId)                         : 10
   Loc: [tst_db.cpp(121)]
QWARN  : TestDB::test_createProgram(Short sysex) QSqlQuery::exec: database not open
QWARN  : TestDB::test_createProgram(Short sysex) Failed to initialize db ""
QWARN  : TestDB::test_createProgram(Short sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Short sysex) Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Short sysex) Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Short sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Short sysex) Failed to retrieve program -1 name: "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Short sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Short sysex) Failed to generate sysex for program -1 : "Driver not loaded Driver not loaded"
INFO   : TestDB::test_createProgram(Short sysex) Did not receive message: "Failed to update program from sysex:  " ""
INFO   : TestDB::test_createProgram(Short sysex) Did not receive message: "Failed to generate sysex for program -1 : " ""
FAIL!  : TestDB::test_createProgram(Short sysex) Not all expected messages were received
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) QSqlQuery::exec: database not open
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) Failed to initialize db ""
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) Failed to retrieve program -1 name: "Driver not loaded Driver not loaded"
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) QSqlQuery::prepare: database not open
QWARN  : TestDB::test_createProgram(Out of bound value in sysex) Failed to generate sysex for program -1 : "Driver not loaded Driver not loaded"
INFO   : TestDB::test_createProgram(Out of bound value in sysex) Did not receive message: "Failed to update program from sysex:  "CHECK constraint failed: programs Unable to fetch row""
INFO   : TestDB::test_createProgram(Out of bound value in sysex) Did not receive message: "Failed to generate sysex for program -1 : " ""
FAIL!  : TestDB::test_createProgram(Out of bound value in sysex) Not all expected messages were received
PASS   : TestDB::cleanupTestCase()
Totals: 2 passed, 6 failed, 0 skipped, 0 blacklisted, 53ms
********* Finished testing of TestDB *********
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_CreateProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_CreateProgramCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
FAIL!  : TestCommands::test_CreateProgramCommand() Compared lists have different sizes.
   Actual   (programIds()) size: 0
   Expected (QList<int>() << 1 << 2 << 3) size: 3
   Loc: [tst_commands.cpp(51)]
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::exec: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to initialize db ""
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to retrieve program 2 name: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to generate sysex for program 2 : "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to delete program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_DeleteProgramCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_DeleteProgramCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
FAIL!  : TestCommands::test_DeleteProgramCommand() Compared lists have different sizes.
   Actual   (programIds()) size: 0
   Expected (QList<int>() << 1 << 3) size: 2
   Loc: [tst_commands.cpp(67)]
QWARN  : TestCommands::test_UpdateParameterCommand() QSqlQuery::exec: database not open
QWARN  : TestCommands::test_UpdateParameterCommand() Failed to initialize db ""
QWARN  : TestCommands::test_UpdateParameterCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateParameterCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
PASS   : TestCommands::test_UpdateParameterCommand()
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::exec: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to initialize db ""
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to retrieve a list of program ids: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to create program:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to commit program creation transaction: "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to generate sysex for program -1 : "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to generate sysex for program -1 : "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to update program from sysex:  "Driver not loaded Driver not loaded"
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() QSqlQuery::prepare: database not open
QWARN  : TestCommands::test_UpdateProgramFromSysexCommand() Failed to generate sysex for program -1 : "Driver not loaded Driver not loaded"
FAIL!  : TestCommands::test_UpdateProgramFromSysexCommand() Compared values are not the same
   Actual   (programSysex(id)): ""
   Expected (updated)         : "G\x00o&/\x00""A\x00m&\x00\x00""E\x00m&-\x00\x00\x00\x00\x00"
   Loc: [tst_commands.cpp(100)]
PASS   : TestCommands::cleanupTestCase()
Totals: 3 passed, 3 failed, 0 skipped, 0 blacklisted, 88ms
********* Finished testing of TestCommands *********
Makefile:313: recipe for target 'check' failed
make[3]: *** [check] Error 6
make[3]: Leaving directory '/build/lpd8editor-0.0.12/test/TestDB'
Makefile:343: recipe for target 'sub-TestDB-check' failed
make[2]: *** [sub-TestDB-check] Error 2
make[2]: *** Waiting for unfinished jobs....
Makefile:313: recipe for target 'check' failed
make[3]: *** [check] Error 3
make[3]: Leaving directory '/build/lpd8editor-0.0.12/test/TestCommands'
Makefile:346: recipe for target 'sub-TestCommands-check' failed
make[2]: *** [sub-TestCommands-check] Error 2
make[2]: Leaving directory '/build/lpd8editor-0.0.12/test'
Makefile:219: recipe for target 'sub-test-check' failed
make[1]: *** [sub-test-check] Error 2
make[1]: Leaving directory '/build/lpd8editor-0.0.12'
dh_auto_test: make -j2 check returned exit code 2
debian/rules:7: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
I: copying local configuration
E: Failed autobuilding of package
```

Ça te parle ?

--- a/debian/control
+++ b/debian/control
@@ -5,6 +5,7 @@ Maintainer: Équipe LibraZiK <librazik@tuxfamily.org>
 Uploaders: Olivier Humbert <trebmuh@tuxfamily.org>
 Build-Depends: debhelper (>= 10),
                libasound2-dev,
+               libqt5sql5-sqlite,
                libqt5svg5-dev,
                pkg-config,
                qtbase5-dev,

